notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    
</pre><p>I also got deprecation warnings when I ran this part:</p><p>WARNING:ragas.metrics._context_precision:Using 'context_precision' without ground truth will be soon depreciated. Use 'context_utilization' instead</p><p> </p><p> </p><p>Would you please check if these can be resolved?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I tried resolving this, but wasn't able to.</p>

Reply:
Looks like this can't be resolved because it comes from the ragas code itself. Analysis below.

The warning comes from this [line](https://github.com/explodinggradients/ragas/blob/5050742fc8a5c4ba80e6f899dea87de1f64601c6/src/ragas/metrics/_context_precision.py#L98) in the ragas code. 

- There's not a lot inside `ContextUtilization` [class](https://github.com/explodinggradients/ragas/blob/5050742fc8a5c4ba80e6f899dea87de1f64601c6/src/ragas/metrics/_context_precision.py#L177). It just inherits from `ContextPrecision` class. 
- The functionalities of `ContextUtilization` and `ContextPrecision` are differentiated based on whether ground truth answers are provided.
- Inside `ContextPrecision`, [`_get_row_attributes()`](https://github.com/explodinggradients/ragas/blob/5050742fc8a5c4ba80e6f899dea87de1f64601c6/src/ragas/metrics/_context_precision.py#L94) returns
      -  either row["question"], row["contexts"], row["**answer**"] for context utilization
      -  or returns row["question"], row["contexts"], row["**ground truth**"] for context precision
- Since no `ground truth answers` are provided for context utilization, the above warning is issued.